### PR TITLE
SOF 1909/Show Segment numbers

### PR DESCRIPTION
**Pull request by RasmusAlbrektsen**
This PR handles the `referenceTag` put forth from Sofie Server in PR: https://github.com/tv2/sofie-server/pull/143. It looks like this:
image

### CODE REVIEW

**Review by LindvedKrvang — APPROVED** [review]

File: src/app/rundown-view/components/segment/segment.component.html [hunk 1]
```diff
@@ -7,8 +7,11 @@
     <div class="c-segment__header-toolbar"
       [class.c-segment__header-toolbar--unsynced]="segment.isUnsynced"
     >
-      <span class="segment-identifier">
-        <ng-container *ngIf="segment.isUnsynced">UNSYNCED</ng-container>
+      <span class="segment-reference-tag">
+        <span *ngIf="!segment.isUnsynced else unsynced">{{segment.referenceTag}}</span>
```
Comment (KvelaGorrrrnio) [inline src/app/rundown-view/components/segment/segment.component.html]: The `span` can be a `ng-container` to avoid having the extra element.

Comment (KvelaGorrrrnio) [inline src/app/rundown-view/components/segment/segment.component.html]: Shouldn't the reference tag and unsynced information be available at the same time if both are present?

**Comment by RasmusAlbrektsen** [comment]
This is now the look of the unsynced segments:
image

File: src/app/rundown-view/components/segment/segment.component.html (hunk 1 above)
Comment (RasmusAlbrektsen) [reply src/app/rundown-view/components/segment/segment.component.html]: It is with the new design!

Comment (RasmusAlbrektsen) [reply src/app/rundown-view/components/segment/segment.component.html]: No longer there after the changes.

**Review by KvelaGorrrrnio — APPROVED** [review]

**Pull request merged**